packages/models/src/crypto-assets/crypto-asset-info.model.ts

```diff
@@ -1,68 +1,122 @@
 import { InscriptionMimeType } from './bitcoin/inscription.model';
 
+export const CryptoAssetChains = {
+  bitcoin: 'bitcoin',
+  stacks: 'stacks',
+} as const;
```

Comment:
```suggestion
export const CryptoAssetChains = ['bitcoin', 'stacks'] as const;
export type CryptoAssetChain = (typeof CryptoAssetChains)[number];
```

Any reason these are objects vs just an array?

Reply:
When you define it with array or a literal union like that then it's difficult to satisfy the type when instantiating an asset object - Just using string `"bitcoin"` doesn't satisfy `CryptoAssetChain`. You need to use something like `asset = { chain: "bitcoin" as const, ... }`.